types/hooks/screens/useAnonymousChannelListScreenHook.types.ts

```diff
@@ -1,10 +1,11 @@
 import {ChannelList} from '../../database/schema/ChannelList.types';
+import {ContactScreenPayload} from './useChatUtilsHook.types';
 
 interface UseAnonymousChannelListScreenHook {
   channels: ChannelList[];
   goToChatScreen: (channel: ChannelList) => void;
   goToPostDetailScreen: (channel: ChannelList) => void;
-  goToContactScreen: () => void;
+  goToContactScreen: ({from}: ContactScreenPayload) => void;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function signature for `goToContactScreen` has been updated to accept an object with a `from` property of type `ContactScreenPayload`. This is a good change as it allows the function to be more flexible and adaptable to different use cases. However, please ensure that all calls to this function throughout the codebase have been updated to match this new signature.
```diff
-  goToContactScreen: () => void;
+  goToContactScreen: ({from}: ContactScreenPayload) => void;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->